src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -8,6 +8,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
+    <!-- We do not want to block reflection for this assembly -->
+    <BlockReflectionAttribute Condition="'$(TargetGroup)'=='uapaot'">false</BlockReflectionAttribute>
```

Comment:
If there's such a thing as a type-by-type whitelist in corefx, I don't know where it lives. From what I've seen, framework libraries in corefx just get the ReflectionBlock attribute definition injected into them - after that, it's the visibility access that controls blocking. The only way to unblock a non-api type/member is to change its visibility to "public"